modules/openapi-generator/src/main/resources/python/configuration.mustache

```diff
@@ -143,14 +143,7 @@ class Configuration(object):
 
     @property
     def logger_file(self):
-        """The logger file.
-
-        If the logger_file is None, then add stream handler and remove file
-        handler. Otherwise, add file handler and remove stream handler.
-
-        :param value: The logger_file path.
-        :type: str
-        """
```

Comment:
Why are we removing this comment?
It provides useful info about the type of this property and how to use it

Reply:
But this is a getter not setter. The signature is wrong.

Reply:
Please add these docstrings back in.
When one uses help to find out how to use the methods of a class, it only gives back the docstring in the getter per:
https://stackoverflow.com/questions/16025462/what-is-the-right-way-to-put-a-docstring-on-python-property
Also having the getter describe the type returned by the getter his helpful to users

Reply:
If the reason is that the setter docstrings are ignored then I've removed them and adjusted the getter ones.